packages/configs/default/index.json

```
@@ -60,6 +62,7 @@
     "*.css": "@parcel/packager-css",
     "*.js": "@parcel/packager-js",
     "*.ts": "@parcel/packager-ts",
+    "*.json": "@parcel/packager-json",
     "*.jsonld": "@parcel/packager-raw-url",
```

```suggestion
    "*.{json,jsonld}": "@parcel/packager-raw-url",
```

I don't see what your json packager does differently compared to that one

---

Removes absolute paths and uses relative ones only